components/cli/pkg/commands/test.go

```diff
@@ -180,6 +180,9 @@ func RunTest(cellImageTag string, instanceName string, startDependencies bool, s
 	if err != nil {
 		util.ExitWithErrorMessage("Failed to test Cell instance"+instanceName, err)
 	}
+	//Remove Ballerina toml and local repo to avoid failing cellery build and run
+	os.Remove(constants.BALLERINA_TOML)
+	os.RemoveAll(constants.BALLERINA_LOCAL_REPO)
```

Comment:
Do we need to check whether the .ballerina is created by cellery test before we delete it? Because there is .ballerina in the home directory as well, so I think we are not supposed to delete that directory. We only have to delete the .ballerina folder created during the test execution.

Reply:
Btw, is this directories are getting created in the tmp location? Not in directory where the command is executed?

Reply:
A target folder is created in the pwd for running tests. So it will clean the .ballerina and Ballerina toml that is in the pwd/targed directory

Reply:
Ok, I somehow experienced a situation where .ballerina was created in pwd, not in pwd/target. If it's creating to pwd/target, then it's safe to add this fix.